### DISCUSSION TITLE
Combo chart - with line and bar

### DISCUSSION THREAD

**Discussion by fulvio80** [imaNNeo/fl_chart#1171] [Q&A]
hello guys,
I would like to use fl_chart to make a combo chart: line and bar, just like in the image. I often use fl_chart but I have never had this need yet. Do you have any suggestions? I wouldn't want to use stack, and I'd prefer a good internal practice to fl_chart but I'm not finding the correct way. 
Tnx
![image](https://user-images.githubusercontent.com/91665947/195886125-7f831880-f363-4ae5-81f6-20113bb2d22b.png)

**Comment by imaNNeo** [comment]
There is an issue for this feature, you can follow it up and add a thumbs up
#764
